denizen_scripts/survival/repo_link/systems/vote_handlers.dsc

```diff
@@ -0,0 +1,151 @@
+vote_reciever:
+  type: world
+  debug: false
+  events:
+    #Vote Recieved:
+      on votifier vote:
+      - define listed_sites <element[to_be_filled_in_when_listed_on_sites]>
```

Comment:
there are two references to this definition but they are checking it as if it were a number, which makes me believe this was meant to be a list
```suggestion
      - define listed_sites <list[to_be_filled_in_when_listed_on_sites]>
```

Reply:
Its meant to have a number filled in, as its used for math later. Its just an easier way to store how many websites we list on for votifier votes.